scheduler/src/cook/mesos/monitor.clj

```diff
@@ -155,14 +157,15 @@
   "Starts a periodic timer to collect stats about running, waiting, and starved jobs per user.
 
    Return a function which can be used to stop collecting stats if invoked."
-  [mesos-conn interval-seconds]
```

Comment:
Yay! 

Can we add a unit test for this to see how the code looks like when mocking `mount` initialized `def`s?

Reply:
Done, I added `test-start-collecting-stats`, let me know what you think